src/vaultRegistry/vault/metadata/strategyMetadata.ts

```diff
@@ -0,0 +1,95 @@
+export type StrategyMetadata = { [key: string]: { name: string, description: string } }
+
+
+const StrategyBaseMetadata: StrategyMetadata = {
+  yearnUsdcStrategies: {
+    name: "Yearn Compound Folding",
+    description: `**Compound Folding** \- Supplies and borrows USDC on Compound Finance simultaneously to earn COMP. Flashmints are then used to mint DAI from MakerDAO to flashlend and fold the position to boost APY. Earned tokens are then harvested, sold for more USDC, and then deposited back into the strategy.
+    ----
+      **Idle Finance Reinvest** \- Supplies USDC to Idle Finance to earn IDLE and COMP. Earned tokens are harvested, sold for more USDC, and then deposited back into the strategy.
+      ----
+      **Angle Reinvest** \- Provides USDC liquidity to Angle Protocol for sanTokens that are staked to earn ANGLE. Earned tokens are harvested, sold for more USDC, and deposited back into the strategy.
+      ----
+      **Maker Folding** \- Supplies USDC to MakerDAO Peg Stability Module for a USDC-DAI ratio that is then deposited int the Uniswap v2 DAI-USDC liquidity pool. Flashmints are used to mint DAI from MakerDAO to flashlend and fold the position, boosting the APY. Earned tokens are harvested, sold for more USDC, and then deposited back into the strategy.`
+  },
+  fluxLending: {
+    name: "Flux Lending",
+    description: "Supplies assets into Flux Finance to earn lending interest. These assets get borrowed by accredited investors which supply on-chain US Treasuries as collateral."
+  },
+  ousd: {
+    name: "OUSD",
+    description: `OUSD integrates with Aave and Compound to automate yield on over-collateralized loans.
+    ----
+    The OUSD protocol also routes USDT, USDC, and DAI to highly-performing liquidity pools as determined by trading volume and rewards tokens (e.g. Curve rewards CRV tokens to liquidity providers). Yields are then passed on to OUSD holders.
+    ---
+    In addition to collecting interest from lending and fees from market making, the protocol automatically claims and converts bonus incentives that are being distributed by DeFi protocols.`
+  },
+  oeth: {
+    name: "OETH",
+    description: `OETH integrates with various Liquid Staking Provider to optimize interest earned by staking Ether.
+    ----
+    The OETH protocol also utilizes Curve and Convex Finance to earn trading fees and additional rewards on ETH / OETH. It automatically claims and converts bonus incentives that are being distributed by these protocols.`
+  },
+  stYCrv: {
+    name: "Staked yCRV",
+    description: "Accepts yCRV to earn a continuous share of Curve Finance fees and Curve DAO voting bribes. Earned 3Crv (Curve\'s 3pool LP token) fees and rewards are harvested, swapped for more yCRV which is deposited back into the strategy. Swap happens either via market-buy or mint, depending which is more capital efficient."
+  },
+
+}
```

Comment:
should this really be part of the SDK? That's stuff that belongs into the frontend imo.